README.md

```diff
@@ -42,13 +42,13 @@ To plot the output as a heatmap:
 
 # Develop
 
-Continually build and test while developing. This will automatically create your virtual env
+The C code (`pyFlowSOM/flowsom.c`) is wrapped using Cython (`pyFlowSOM/cyFlowSOM.c`).
```

Comment:
This is pretty insignificant, but I think the cython file is actually ".pyx"? So the comment should be:
The C code (`pyFlowSOM/flowsom.c`) is wrapped using Cython (`pyFlowSOM/cyFlowSOM.pyx`).

Reply:
Yeah good catch, looks like an old typo.